tests/integration/hive_metastore/test_migrate.py

```diff
@@ -101,3 +102,44 @@ def test_migrate_external_table(ws, sql_backend, inventory_schema, make_catalog,
 
     target_tables = list(sql_backend.fetch(f"SHOW TABLES IN {dst_schema.full_name}"))
     assert len(target_tables) == 1
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_revert_migrated_table(ws, sql_backend, inventory_schema, make_schema, make_table):
+    schema1 = make_schema(catalog_name="hive_metastore")
+    schema2 = make_schema(catalog_name="hive_metastore")
+    tables = [
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema2.name),
+        make_table(schema_name=schema2.name),
+    ]
+
+    for table in tables[0:3]:
+        # apply "upgraded to" table property to the first three tables.
+        logger.info(f"Applying upgraded_to table property for {table.full_name}")
+        sql_backend.execute(f"ALTER TABLE {table.full_name} set TBLPROPERTIES('upgraded_to'='FAKE DEST')")
```

Comment:
Please invoke the real method from TablesMigrate, not the fake sql statement

Reply:
Ah... so physically migrate the tables.

Reply:
Fixed